### DISCUSSION TITLE
why don't package version found with npm: specifier match the version on npm site?

### DISCUSSION THREAD

**Discussion by bkerin** [denoland/deno#24254] [General]
If I say:
```
import { BetterSqlite3Adapter } from "npm:lucia-auth/adapter-sqlite"
```
then poke around in ~/.cache/deno/npm I find that the version installed is 1.8.0, so I can say:
```
import { BetterSqlite3Adapter } from "npm:lucia-auth@1.8.0/adapter-sqlite"
```
and it works.  The odd thing is that 1.8. doesn't even show up in the version history on https://www.npmjs.com/package/@lucia-auth/adapter-sqlite.
The current version there appears to be 3.0.1, but using that in the npm: specifier result in a "could not find package" error.

Is there some description somewhere of where the version in npm: specifiers are coming from?

**Comment by dsherret** [comment]
Note that the link you provided is for the `@lucia-auth/adapter-sqlite` and what's being imported is `lucia-auth`, which has a deprecated 1.8.0 version that indeed exists: https://www.npmjs.com/package/lucia-auth?activeTab=versions
